libs/gson-support/src/main/java/org/codice/gsonsupport/GsonTypeAdapters.java

```diff
@@ -46,7 +47,8 @@ private GsonTypeAdapters() {}
   public static class DateLongFormatTypeAdapter extends TypeAdapter<Date> {
     @Override
     public void write(JsonWriter out, Date date) throws IOException {
-      out.value(date.getTime());
+      Long time = Optional.ofNullable(date).map(Date::getTime).orElse(null);
```

Comment:
Is there a unit test we can add for this?

Reply:
Added some basic ones around this issue.